Exiled.Events/Handlers/Player.cs

```diff
@@ -465,6 +465,12 @@ public class Player
         /// <param name="ev">The <see cref="ReservedSlotsCheckEventArgs"/> instance.</param>
         public static void OnReservedSlot(ReservedSlotsCheckEventArgs ev) => ReservedSlot.InvokeSafely(ev);
 
+        /// <summary>
+        /// Called before reserved slot is resolved for a <see cref="API.Features.Player"/>.
+        /// </summary>
+        /// <param name="ev">The <see cref="ReservedSlotsCheckEventArgs"/> instance.</param>
+        public static void OnReservedSlot(ReservedSlotsCheckEventArgs ev) => ReservedSlot.InvokeSafely(ev);
```

Comment:
This event is duplicated.

Reply:
apparently someone commit it in my branch ^^'

Reply:
Done